test/jdk/java/net/httpclient/http2/TrailingHeadersTest.java

```diff
@@ -0,0 +1,217 @@
+/*
+ * Copyright (c) 2017, 2023, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Hello Conor, this appears to be a typo. Given that this is a new file, only 2023 would be relevant.

Reply:
I'll hold off on updating this until more discussion takes place below, for ease of VC use locally